### ISSUE TITLE
Improve current profile page

### ISSUE THREAD

**Issue by Grizzly127** [Kompakkt/Repo#59]
### improve the current profile page to introduce more features 

**Add selection option for the current profile page** 
Here the first quick ideas for improving the current profile page. (I made it in goodnotes, that’s why there is this notebook page first, I didn’t know how to delete it) on the first two pages the user sees his own profile. We want to add a select objects button here and replace Groups with Collaborations/Projects.
Once the user clicks ob select objects, the objects get tickboxes. He can choose objects individually or click “select all” on top. While selecting the three dots appear, this is where the manage options to delete, publish or add to collection appear.
The third page is the page, how a user could see the profile of another user or an institution. The profile name is the title, then we can add some information, then the public objects of this person are listet, then the collections (I thought maybe collections before Collaborations could make sense, to have objects and collections closer together. If it makes sense, we change it at the sites before as well).
And as a last point there are Collaborations of the user. I put the kompakkt sign in front to make it clearer that it is another profile where you can go to. I’ll mock it up as an institutional profile as well.

<img width="836" alt="Bildschirmfoto 2024-12-11 um 15 20 25" src="https://github.com/user-attachments/assets/77639205-7839-4129-9650-9172072ea453" />
<img width="842" alt="Bildschirmfoto 2024-12-11 um 15 20 32" src="https://github.com/user-attachments/assets/d409025c-11ef-4271-acc8-26fa790d97f7" />
<img width="849" alt="Bildschirmfoto 2024-12-11 um 15 25 27" src="https://github.com/user-attachments/assets/deb03ae9-50ac-41d2-bad2-cd6add198031" />

**profile page as an institutional page**
okay, so for institutions it’s the same. There is the name of the institution. Which information do we want to have to them in the second section?
Then their objects and collections and then the Collaborators of this institution/project.
<img width="864" alt="Bildschirmfoto 2024-12-12 um 10 26 31" src="https://github.com/user-attachments/assets/121317aa-5d66-4ef2-9a34-87b37dffb2ee" />

@ZetOE @lozanaross

**Comment by Grizzly127** [comment]
### Google drive inspiration

Using google drive as an inspiration, they didn't use the boxes to mark an object, but the usual keyboard functions: you can select an object by clicking on it and by holding the select or command key you can choose more than one object. 
With the three dots next to the object you can see the options menu. 

<img width="359" alt="Image" src="https://github.com/user-attachments/assets/bf28e3c2-10d8-4d10-afcf-e0a76d818043" />

On top the user can see how many objects are selected. There are also the three dots with the options menu, but the options that can't be assigned to multiple objects are greyed out. 
And some are not shown here, why are some options greyed out and some are not shown here? Where is the difference? 

<img width="813" alt="Image" src="https://github.com/user-attachments/assets/edd74273-f9ae-4147-8319-7afc07289061" />

**Comment by Grizzly127** [comment]
I adapted the google drive functionality to kompakkt: 

Clicking at the three dots for an object, you get the options menu here
<img width="702" alt="Image" src="https://github.com/user-attachments/assets/9e266d2e-6be5-4e95-9be5-7c394aa007e5" />

When more objects are selected, they are marked in blue and on top the options bar appears. Either only with three dots and there are options for multiple objects or some options are already displayed with icons like in google drive: 

<img width="666" alt="Image" src="https://github.com/user-attachments/assets/e1a3cd28-8123-4f3e-a606-8f0ea2591010" />

<img width="799" alt="Image" src="https://github.com/user-attachments/assets/46c6c9b9-b848-43c1-9b5d-423affd2cfc5" />

@ZetOE @lozanaross

**Comment by vmalieske** [comment]
Current update: mostly done, waiting for visibility and access part on multiselection.

ToDo:
- Role-Funktionality on Profile-Page (Viewer, Owner, Editor) --> restricted objects
- Display Visibility and access for multiselection
- Download-Option for objects (multi and single) --> waiting for updated datamodel in backend 
- Publish-option (multi)

**Comment by vmalieske** [comment]
> the multiselection for the current profile page is done now, right? [@vmalieske](https://github.com/vmalieske)

Yes, multiselection for the profile page is done :) fixes or bugs could be in separate issues from now on, I guess.

**Comment by Grizzly127** [comment]
okay, great, then I close this issue.
